pkg/config/server/server.go

```diff
@@ -218,6 +218,9 @@ type SentryConfigFile struct {
 
 	// Environment is the environment that the instance is running in
 	Environment string `mapstructure:"environment" json:"environment,omitempty" default:"development"`
+
+	// Sample rate is the rate at which to sample events. Default is 1.0 to sample all events.
+	SampleRate float64 `mapstructure:"sampleRate" json:"sampleRate,omitempty" default:"1.0"`
```

Comment:
should we default to lower? i dont have a strong opinion but cc @abelanger5 since we'll need to set this on all instances if we want to lower sampling

Reply:
my thought was that if people are self-hosting and using sentry now and we default to lower, they'll see our sampling logic kick in without them knowing to set this env var. but not 100% sure this is the right path 🤷

Reply:
good call. https://docs.hatchet.run/self-hosting/configuration-options#alerting-configuration

we are publishing sentry, i thought we were not.

let's remember to set this in the next release

Reply:
sounds good! do I need to update this on the docs site? I didn't think to look there

Reply:
nice to have

Reply:
ticket: https://linear.app/hatchet-dev/issue/HAT-323/documentation-on-sentry-sampling